docker-compose-dev-linux.yml

```diff
@@ -0,0 +1,23 @@
+version: '3'
```

Comment:
should probably rename the other file to `-mac`?

Reply:
And update the readme for that.

Reply:
Making a seperate PR